script/Arcueid/Day 2/02_00_ARC02_1_1.txt

```diff
@@ -259,7 +259,7 @@ I casually check the anonymous bulletin board for students.
 [sha:c3a878b55b2805d7e460bb220454c47639b37880]{
 -- Page 142, Offset 2975.
 -- 『っていうか、あのオンナもちょっと露骨じゃねえ？』
-'I mean, isn't that woman a little too blatant?'
+'I mean, isn't she a little too shameless?'
```

Comment:
dunno where "too" is coming from. This should just be "I mean, isn't she a little shameless?"